packages/kit/src/context.ts

```diff
@@ -1,40 +1,42 @@
 import { getContext } from 'unctx'
-import type { Nuxt } from '@nuxt/schema'
+import { type Nuxt } from '@nuxt/schema'
 
 /** Direct access to the Nuxt context - see https://github.com/unjs/unctx. */
-export const nuxtCtx = getContext<Nuxt>('nuxt')
-
-// TODO: Use use/tryUse from unctx. https://github.com/unjs/unctx/issues/6
+export const nuxtContext = getContext<Nuxt>('nuxt')
```

Comment:
We likely need to keep this export the same or it is a breaking change.

Reply:
Added `nuxtCtx` to the allowlist of `unicorn/prevent-abbreviations`.